crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -500,7 +567,10 @@ mod tests {
         let (target_dir, data_transfer) =
             create_data_transfer_component(metadata_manager, temp_dir.path()).await;
 
-        data_transfer.flush().await.unwrap();
+        data_transfer
```

Comment:
Unclear why `data_transfer.flush()` was replaced with`transfer_larger_than_threshold(0)` maybe add a comment or a `flush()` method to `data_transfer()` that is simply executes `transfer_larger_than_threshold(0)` and use that everywhere for consistency with the other storage components as they implements `flush()`.

Reply:
I think it is overkill to provide such methods in components such as the data transfer component since it quite deep in the storage engine architecture and not accessed directly outside of it.